wordpress-mobile/WordPress-Android
# Updated location of stats and page QS focus point.

Fixes #12262 

This PR moves the Quick Start focus point for Stats and Pages from rows to IA buttons. I also renamed the "Tap Site Pages.." in action prompt to just "Tap Pages..."

@osullivanchris I took a little liberty with the placement of a focus point - please let me know if you want it in another place (I can't guarantee precise placement, but can get it pretty close).

[![Image from Gyazo](https://i.gyazo.com/7ecd08c2d8afa646669c445858b5de1c.png)](https://gyazo.com/7ecd08c2d8afa646669c445858b5de1c)

To test:
- Create new site and start a QuickStart.
- Navigate to Customize Your Site task list and pick Create A New Page task.
- Confirm that the focus point is at the Pages IA button.

- Navigate to Grow Your Audience task list and pick Check Your Site Stats task.
- Confirm that the focus point is at the Stats IA button.


PR submission checklist:
